lib/api/bucketDeleteEncryption.js

```diff
@@ -42,6 +42,11 @@ function bucketDeleteEncryption(authInfo, request, log, callback) {
                 configuredMasterKeyId: sseConfig.configuredMasterKeyId,
```

Comment:
can we directly add it here?
```
isAccountEncryptionEnabled: sseConfig.isAccountEncryptionEnabled,
```

Reply:
It is just to avoid storing the property when it is not needed.

Reply:
Okay, I am good if its an intentional choice.